pg_metadata/install/sql/upgrade/upgrade_to_1.1.0.sql

```diff
@@ -330,4 +329,110 @@ BEGIN
 END;
 $$;
 
+
```

Comment:
You need to move this content in a new file 1.2.0
1.1.0 is out : https://github.com/3liz/qgis-pgmetadata-plugin/releases/tag/1.1.0

You should rebase your branch on top of master.

Reply:
Sorry, when I prepared the PR I forgot that 1.1.0 is already published. I’ll fix it soon.

However, I thought I had rebased on current master.